### PR TITLE
manifest.json

### PR DESCRIPTION
**Pull request by limhenry**
TODO: service worker for "Add to Homescreen" banner and offline.

Note: Icon designed by me.

### CODE REVIEW

**Comment by cheeaun** [comment]
Would be cool if got a screenshot of how it looks like on a phone (Nexus?)

**Comment by limhenry** [comment]
**Screenshot:** https://goo.gl/photos/M978D1Avk3pXnjFL6

**Video:** 
[https://www.youtube.com/watch?v=A66vZ2nAJrU](https://www.youtube.com/watch?v=A66vZ2nAJrU) 

First Part:
 - https://cheeaun.github.io/repokemon/ 

Second Part:
 - Repokemon with manifest.json (built with polymer)


_note: i have no idea why some images are not showing_

**Comment by cheeaun** [comment]
> note: i have no idea why some images are not showing

you mean it doesn't show after adding the manifest or before?

**Comment by limhenry** [comment]
No. That's your website. It's not on localhost.

*btw, I think you should serve the image using HTTPS

**Comment by cheeaun** [comment]
Hmm already https now, is it still not loading? @limhenry 

For me, all images load but slow because connection limits in the browser 😏

File: manifest.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Repokémon",
+  "short_name": "Repokémon",
+  "icons": [{
+        "src": "/pokeball.png",
+        "sizes": "512x512",
+        "type": "image/png"
+      }],
+  "start_url": "/",
```
Comment (cheeaun) [inline manifest.json]: hmm does this go to root? 🤔

**Comment by cheeaun** [comment]
Would be cool if you can commit the SVG version also. I'll credit you in README, or you could include yourself in the README :P

File: manifest.json (hunk 1 above)
Comment (limhenry) [inline manifest.json]: yes :sob:

**Comment by limhenry** [comment]
sure thing. i will make a pr later

**Comment by limhenry** [comment]
@cheeaun done.

updated the `start_url` and added a super nice pokeball svg file. :laughing:

**Comment by limhenry** [comment]
sh*t. i forgot to credit myself...

**Comment by cheeaun** [comment]
@limhenry can do another PR :P

**Pull request merged**